dev-docs/adding-data/adding_test_data.md

```diff
@@ -133,8 +133,15 @@ function prepare_chef_load() {
 `source .studio/chef-load`
 
 ## Adding data to event feed
-follow the instructions for "Modifying the hab studio functions to talk to another automate url"
-`chef_load_actions`
+follow the instructions for "Modifying the hab studio functions to talk to another automate url" then: 
+```
+chef_load_actions
+```
+
+Troubleshooting If your event feed guitar string graph does not show up locally in the UI run:
```

Comment:
```suggestion
Troubleshooting: If your event feed guitar string graph does not show up locally in the UI run:
```

Reply:
Good catch - flaky tests are breaking, this took 3 hours for me to get these small changes to pass. So I am gonna merge this in and then I’ll add that in the next iteration when I have time to watch it.